### PR TITLE
Update p2pool-dash for working with DashCore v0.12.1.x

### PR DESCRIPTION
**Pull request by thelazier**
Please let me know if we need to add anything.
Possibly version bump to compatible with DashCore release.

### CODE REVIEW

**Comment by thelazier** [comment]
BTW, p2pool-dash seed nodes possibly need to update, suggestions are welcome.

File: p2pool/dash/helper.py [hunk 1]
```diff
@@ -14,12 +14,12 @@ def check(dashd, net):
         print >>sys.stderr, "    Check failed! Make sure that you're connected to the right dashd with --dashd-rpc-port!"
         raise deferral.RetrySilentlyException()
     if not net.VERSION_CHECK((yield dashd.rpc_getinfo())['version']):
-        print >>sys.stderr, '    dash version too old! Upgrade to 0.11.2.17 or newer!'
+        print >>sys.stderr, '    dash version too old! Upgrade to 0.12.0.58 or newer!'
```
Comment (UdjinM6) [inline p2pool/dash/helper.py]: 0.12.1.0

File: p2pool/dash/p2p.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def __init__(self, net):
 
     def connectionMade(self):
         self.send_version(
-            version=70077,
+            version=70204,
```
Comment (UdjinM6) [inline p2pool/dash/p2p.py]: 70206

**Comment by thelazier** [comment]
Tested on testnet , I'm running a mainnet node (p2pool.dash.siampm.com), we still need to get in touch with other pools to update seed nodes.

**Comment by UdjinM6** [comment]
Another user confirmed successful node start with this code https://www.dash.org/forum/threads/p2pool-repository-and-python-modules-may-2016.8795/#post-113168

Probably ok to merge. @schinzelh ?

**Comment by UdjinM6** [comment]
Post-merge update:
- Confirmed by another user https://www.dash.org/forum/threads/p2pool-repository-and-python-modules-may-2016.8795/#post-113237
- 12.1 block mined by p2pool on mainnet https://explorer.dash.org/block/0000000000004f0ea4964d50b845d62893fa89138ece2cd2630b48e13ba66661

**Pull request merged**